### PR TITLE
Flekschas/hotfix fragments api cool v2

### PR DESCRIPTION
**Pull request by flekschas**
## Description

> What was changed in this pull request?

- Fixed the way cooler is loaded
- Fixed a variable clash that somehow arose at some point

Why is it necessary?

Loading snippets from mcool v2 files failed in some cases

## Checklist

- [x] Unit tests added or updated
- [ ] Documentation added or updated
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: fragments/utils.py [hunk 1]
```diff
@@ -243,6 +245,7 @@ def get_cooler(f, zoomout_level=0):
         pass
```
Comment (nvictus) [inline fragments/utils.py]: why not log this too? linters tend to hate bare `except`s anyway.

File: fragments/utils.py [hunk 2]
```diff
@@ -243,6 +245,7 @@ def get_cooler(f, zoomout_level=0):
...
 
     try:
+        zoomout_level = 0 if zoomout_level is None else zoomout_level
         # v1
         zoom_levels = np.array(list(f.keys()), dtype=int)
```
Comment (nvictus) [inline fragments/utils.py]: redundant `pass` on L265

Comment (flekschas) [reply fragments/utils.py]: Good catch!

File: fragments/utils.py (hunk 1 above)
Comment (flekschas) [reply fragments/utils.py]: Good catch!

**Pull request merged**